### PR TITLE
Extend docs

### LINKED ISSUE #5: Improve documentation
- [ ] describe token usage
- [ ] extend example workflow

### PR DESCRIPTION
**Pull request by TobKed**
fix https://github.com/TobKed/github-forks-sync-action/issues/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,25 @@ jobs:
...
 | force | boolean | false | Determines if force push is used. |
 | tags | boolean | false | Determines if `--tags` is used. |
+
+### GitHub Token
+
+#### Pull from public repository and push to current repository
+
+GitHub automatically creates a `GITHUB_TOKEN` secret to use in your workflow. 
+You can use the `GITHUB_TOKEN` to authenticate in a workflow run. 
+`github_token` input can passed in `${{ secrets.GITHUB_TOKEN }}`
```
Comment (turbaszek) [inline README.md:43]: Is there any documentation about it? If yes should it be added as a reference as it is done below?

**Comment by turbaszek** [comment]
Although it may seem obvious, I would add a short description of this Github action. What it does and when it can be helpful 💪

**Comment by turbaszek** [comment]
Does this action run on every change in repo?

File: README.md [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
 ```yaml
 jobs:
-  build:
+  update_external_airflow_fork:
     runs-on: ubuntu-latest
     steps:
       uses: TobKed/github-forks-sync-action@master
       with:
-        github_token: ${{ secrets.GH_TOKEN }}
+        github_token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
```
Comment (turbaszek) [inline README.md]:
```suggestion
        github_token: ${{ secrets.GITHUB_TOKEN }}
```

File: README.md (hunk 1 above)
Comment (turbaszek) [inline README.md:43]:
```suggestion
GitHub automatically creates a `GITHUB_TOKEN` secret to use in your workflow. 
You can use the `GITHUB_TOKEN` to authenticate in a workflow run. 
To learn more about this secret check the [official docs](https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token).
```

File: README.md [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
 ```yaml
 jobs:
-  build:
+  update_external_airflow_fork:
     runs-on: ubuntu-latest
     steps:
       uses: TobKed/github-forks-sync-action@master
       with:
-        github_token: ${{ secrets.GH_TOKEN }}
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (TobKed) [inline README.md]: @turbaszek I figured out that `secrets.GITHUB_TOKEN` is not a valid one. As the [docs](https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret) says about `GITHUB_TOKEN`:

> The token's permissions are limited to the repository that contains your workflow.

This example is for updating external repository and requires Personal Access Token (https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)

Comment (turbaszek) [reply README.md]: It would be good to mention what privilages should have the secret

**Pull request merged**